client/common/cmdline.h

```diff
@@ -276,8 +284,19 @@ static const COMMAND_LINE_ARGUMENT_A global_cmd_args[] = {
 	  "Use smart card authentication with password as smart card PIN" },
 	{ "pcb", COMMAND_LINE_VALUE_REQUIRED, "<blob>", NULL, NULL, -1, NULL, "Preconnection Blob" },
 	{ "pcid", COMMAND_LINE_VALUE_REQUIRED, "<id>", NULL, NULL, -1, NULL, "Preconnection Id" },
+	{ "card", COMMAND_LINE_VALUE_OPTIONAL, "<card name>", NULL, NULL, -1, NULL, "Card name" },
```

Comment:
Maybe add this as options to `/smartcard:card:aaa,csp:yyy` or `/smartcard-logon:card:aaa` (whichever fits better)?

Reply:
I'd be for keeping these switch with a single option per switch as some of them have values with spaces or `:` (don't wanna have the hell of escaping all that)